src/foraging_gui/Foraging.py

```diff
@@ -2013,7 +2013,13 @@ def _Open(self):
                             elif Tag==1:
                                 index = widget.findText(value)
                             if index != -1:
-                                widget.setCurrentIndex(index)
+                                # Alternating on/off for SessionStartWith if SessionAlternating is on
```

Comment:
So this is alternating whether the session starts in an opto block, or not?

Reply:
This is a very good point. Usually, we performed the optogenetics in consecutive sessions, so alternating the `SessionStartWith` should be fine. However, it's better to always check if this is an opto session. I have updated the code.